app/controllers/mccloud/EnterPstrController.scala

```diff
@@ -101,14 +77,15 @@ class EnterPstrController @Inject()(override val messagesApi: MessagesApi,
           case None => form()
         }
 
-        chargeTitle(chargeType, schemeIndex) match {
-          case Some(title) =>
+        (ordinal(Some(schemeIndex)).map(_.resolve).getOrElse(""), lifetimeOrAnnual(chargeType)) match {
```

Comment:
As before r.e. string, won't repeat comment for future instances as we can just replace all if we decide to do so

Reply:
As per prev comment